File: Basic Coding Problems/02 If - Elses/027 Write a program to input any alphabet and check whether it is vowel or consonant/Java/vowel_consonant.java
import java.util.Scanner;

public class vowel_consonant {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter any alphabet: ");
        char letter = scanner.next().charAt(0);

        if (letter == 'a' || letter == 'e' || letter == 'i' || letter == 'o' || letter == 'u' || letter == 'A' || letter == 'E' || letter == 'I' || letter == 'O' || letter == 'U') {
            System.out.println("The alphabet is a vowel.");
        } else {
            System.out.println("The alphabet is a consonant.");
        }
    }
}

File: Basic Coding Problems/02 If - Elses/027 Write a program to input any alphabet and check whether it is vowel or consonant/C/vowel_consonant.c
#include <stdio.h>

int main() {
    char letter;

    printf("Enter any alphabet: ");
    scanf("%c", &letter);

    if (letter == 'a' || letter == 'e' || letter == 'i' || letter == 'o' || letter == 'u' || letter == 'A' || letter == 'E' || letter == 'I' || letter == 'O' || letter == 'U') {
        printf("The alphabet is a vowel.\n");
    } else {
        printf("The alphabet is a consonant.\n");
    }

    return 0;
}